### PR TITLE
[SPARK-30179][SQL][TESTS] Improve test in SingleSessionSuite

### PR DESCRIPTION
**Pull request by leoluan2009**
### What changes were proposed in this pull request?

improve the temporary functions test in `SingleSessionSuite` by verifying the result in a query

### Why are the changes needed?

Improve test

### Does this PR introduce any user-facing change?

No

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by leoluan2009** [comment]
Retest this please

**Comment by wangyum** [comment]
ok to test

**Comment by maropu** [comment]
Welcome to the community, @leoluan2009. Can you add `[SQL][TESTS]` in the title?

File: sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala [hunk 1]
```diff
@@ -790,6 +790,8 @@ class SingleSessionSuite extends HiveThriftJdbcTest {
         Seq(
           "SET foo=bar",
           s"ADD JAR $jarURL",
+          s"CREATE TABLE test_udtf(key INT, value STRING) USING hive",
```
Comment (maropu) [inline sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala]: nit: drop `s` in the head.

Comment (leoluan2009) [reply sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala]: dropped

**Review by maropu — APPROVED** [review]
cc: @dongjoon-hyun

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Comment by leoluan2009** [comment]
Thanks for your review, @wangyum @maropu @HyukjinKwon @dongjoon-hyun

**Pull request closed without merging**